Lib/test/test_locale.py

```diff
@@ -375,22 +375,24 @@ def setUp(self):
             raise unittest.SkipTest('wcscoll/wcsxfrm have known bugs')
         BaseLocalizedTest.setUp(self)
 
-    @unittest.skipIf(sys.platform.startswith('aix'),
-                     'bpo-29972: broken test on AIX')
-    @unittest.skipIf(
-        is_emscripten or is_wasi,
-        "musl libc issue on Emscripten/WASI, bpo-46390"
-    )
+    # @unittest.skipIf(sys.platform.startswith('aix'),
+    #                  'bpo-29972: broken test on AIX')
+    # @unittest.skipIf(
+    #     is_emscripten or is_wasi,
+    #     "musl libc issue on Emscripten/WASI, bpo-46390"
+    # )
```

Comment:
rather than commenting,
We usually add `@unittest.skip("TODO: RUSTPYTHON")` to make updating tests easier in future